### DISCUSSION TITLE
Provide connection error troubleshooting documentation or reference in the "Connection Error" dialog

### DISCUSSION THREAD

**Discussion by suresh-kandoth** [microsoft/azuredatastudio#22481] [Ideas]
When we attempt to connect to a database server or endpoint, we get the Connection Error dialog. It is a large window with error message at the top and lot of blank white space in the bottom half.

![image](https://user-images.githubusercontent.com/25985186/228081500-008d658b-cdbd-41c4-b847-322e8b330a68.png)

Can we provide Connection error troubleshooting links so the user can look up steps to address the problem or error:

[Troubleshooting connectivity issues and other errors with Azure SQL Database and Azure SQL Managed Instance](https://learn.microsoft.com/en-us/azure/azure-sql/database/troubleshoot-common-errors-issues?view=azuresql)

[Troubleshoot connectivity issues in SQL Server](https://learn.microsoft.com/en-us/troubleshoot/sql/database-engine/connect/resolve-connectivity-errors-overview)

[Connect to a SQL Server virtual machine on Azure](https://learn.microsoft.com/en-us/azure/azure-sql/virtual-machines/windows/ways-to-connect-to-sql)

**Comment by erinstellato-ms** [comment]
Tagging @cheenamalhotra for visibility.

**Comment by cheenamalhotra** [comment]
We are working in this space, stay tuned :)
